Automattic/vip-cli
# Remove dual-path logic for checking import status

## Description
This restores the original single-path logic for checking the status of an import job. (Which is now possible due to the API now returning the job data consistently, regardless of site type

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-sql.js @<id> file.sql` for both site types.
1. Verify import progress is shown and that the imports work.
